ui/js/controllers/bugfiler.js

```diff
@@ -53,6 +53,11 @@ treeherder.controller('BugFilerCtrl', [
                         allFailures[i].shift();
                     }
                 }
+
+                if (allFailures[i][0].search("REFTEST TEST-UNEXPECTED-PASS") >= 0) {
```

Comment:
The search()+replace() can be replaced with a single replace(), eg:
`allFailures[i][0].replace("REFTEST TEST-UNEXPECTED-PASS", "TEST-UNEXPECTED-PASS")`